share/pdf2htmlEX.js.in

```diff
@@ -260,7 +260,24 @@ Viewer.prototype = {
     this.pre_hide_pages();
   },
 
+  initialize_radio_button : function() {
+    var elements = document.getElementsByClassName('ir');
+    
+    for(var i = 0; i < elements.length; i++) {
```

Comment:
Tried it, we can't on an HTMLCollection (http://stackoverflow.com/questions/24266313/why-does-my-call-to-foreach-on-an-array-result-in-uncaught-typeerror-undefined)